server/apps/publish/formatters/aap_sms_formatter_test.py

```diff
@@ -42,4 +43,6 @@ def TestSMSFormatter(self):
 
         self.assertGreater(int(seq), 0)
         self.assertDictEqual(item, {'Category': 'a', 'Priority': 'f', 'Sequence': item['Sequence'], 'ident': '0',
-                                    'Headline': 'This is a test headline', 'StoryText': 'The story body'})
+                                    'Headline': 'This is a test headline',
+                                    'StoryText':
+                                        'The story body<br>call helpline 999 if you are planning to quit smoking'})
```

Comment:
This should not contain <br> as AAPSMSFormatter sets a plain text value to StoryText.